cli/man/splinter.1.md

```diff
@@ -55,6 +55,9 @@ SUBCOMMANDS
 `registry`
 : Provides commands to create and manage Splinter registry information.
 
+`upgrade`
```

Comment:
upgrade should also be added to the SEE ALSO section below

Reply:
Done!